packages/console/src/pages/ConnectorDetails/components/SenderTester/index.tsx

```diff
@@ -56,25 +56,19 @@ const SenderTester = ({ connectorId, connectorType, config, className }: Props)
 
   const onSubmit = handleSubmit(async (formData) => {
     const { sendTo } = formData;
+    const result = arbitraryObjectGuard.safeParse(config);
 
-    try {
-      const configJson = JSON.parse(config) as JSON;
-      const data = { config: configJson, ...(isSms ? { phone: sendTo } : { email: sendTo }) };
-
-      await api
-        .post(`/api/connectors/${connectorId}/test`, {
-          json: data,
-        })
-        .json();
-
-      setShowTooltip(true);
-    } catch (error: unknown) {
-      if (error instanceof SyntaxError) {
-        toast.error(t('connector_details.save_error_json_parse_error'));
-      } else {
-        toast.error(t('errors.unexpected_error'));
```

Comment:
seems that this PR truncated this `unexpected_error` case?

Reply:
The `unexpected_error` is not really needed in this code block. It was initially added as logic wise I thought it would be more strict for this `if-clause`. But technically we can only process the syntax error here, no other error can be caught in here.

And since now the syntax error is handled by zod, I just simply removed the whole try-catch block